docs/how_to_contribute/conventions.md

```diff
@@ -90,6 +90,23 @@ We use `VLOG` messages in our code to show us the progress of the current runnin
       - Output of the proof
       - Output of the verifying key serialization
 
+## Profiling with Perfetto
+
+We are currently using Perfetto for tracing and profiling Tachyon. Perfetto provides two primary macros for tracing events: `TRACE_EVENT()` and `TRACE_EVENT_BEGIN()`|`TRACE_EVENT_END()`.
+
+- **`TRACE_EVENT()`**: Use this macro when you want the trace slice to continue until the scope in which it is defined ends. This is the preferred method whenever possible, as it simplifies the code and ensures the trace slice duration is managed by the scope itself.
+
+- **`TRACE_EVENT_BEGIN()` and `TRACE_EVENT_END()`**: Use these macros when you need to manually specify the beginning and end of a trace slice. This approach is suitable for tracing code segments that are too long or complex for a single scope, where adding a scope would be impractical.
+
+### Conventions
+
+- **Preferred Usage**: Always try to use `TRACE_EVENT()` whenever possible for its simplicity and automatic scope management.
+- **Manual Tracing**: Use `TRACE_EVENT_BEGIN()` and `TRACE_EVENT_END()` for longer or more complex code segments where adding a new scope is not feasible.
```

Comment:
Not to sure what the point of this section is since these preferences are mentioned before in their respective sections.